bootstrapvz/common/tasks/filesystem.py

```diff
@@ -99,6 +99,32 @@ def run(cls, info):
         p_map.root.add_mount(p_map.boot, 'boot')
 
 
+class MountAdditional(Task):
+    description = 'Mounting additional partitions'
+    phase = phases.volume_mounting
+    predecessors = [MountRoot]
+
+    @classmethod
+    def run(cls, info):
+        import os
+        from bootstrapvz.base.fs.partitions.unformatted import UnformattedPartition
+        p_map = info.volume.partition_map
+        partitions = []
+        #  we need to get rid of UnformattedPartition and sort the list of additional partitons in order to mount them correctly
+        for partition in info.volume.partition_map.partitions:
```

Comment:
You can replace this with some nice functional code, making the comment almost obsolete:
```py
is_additional = lambda partition: (
    not isinstance(partition, UnformattedPartition)
    and partition.name not in ["boot", "swap", "root"]
)
name_length = lambda partition: len(partition.name)
for partition in sorted(filter(is_additional, partitions), key=name_length):
    # do stuff
```


for partition_name in sorted(partitions, key=len, reverse=False):